src/test/kotlin/it/unibo/protelis/web/execution/simulated/AlchemistVerticleTest.kt

```diff
@@ -0,0 +1,88 @@
+package it.unibo.protelis.web.execution.simulated
+
+import com.fasterxml.jackson.module.kotlin.registerKotlinModule
+import io.vertx.core.Vertx
+import io.vertx.core.json.JsonObject
+import io.vertx.core.json.jackson.DatabindCodec
+import io.vertx.core.logging.Logger
+import io.vertx.core.logging.LoggerFactory
+import io.vertx.junit5.Checkpoint
+import io.vertx.junit5.Timeout
+import io.vertx.junit5.VertxExtension
+import io.vertx.junit5.VertxTestContext
+import it.unibo.protelis.web.execution.ProtelisUpdateMessage
+import it.unibo.protelis.web.execution.simulated.AlchemistVerticle.Companion.finishedAddress
+import it.unibo.protelis.web.execution.simulated.AlchemistVerticle.Companion.initializedAddress
+import it.unibo.protelis.web.execution.simulated.AlchemistVerticle.Companion.setupAddress
+import it.unibo.protelis.web.execution.simulated.AlchemistVerticle.Companion.stepDoneAddress
+import it.unibo.protelis.web.execution.simulated.AlchemistVerticle.Companion.stopAddress
+import java.util.concurrent.TimeUnit
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+
+@DisplayName("AlchemistVerticle tests with Vert.x TestContext")
+@ExtendWith(VertxExtension::class)
+class AlchemistVerticleTest {
+  private val logger: Logger = LoggerFactory.getLogger(AlchemistVerticleTest::class.java)
+
+  init {
+    DatabindCodec.mapper().registerKotlinModule()
+  }
+
+  /*
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [UndocumentedPublicFunction - [AlchemistVerticle should deploy correctly] at /src/src/test/kotlin/it/unibo/protelis/web/execution/simulated/AlchemistVerticleTest.kt:34:3](https://app.codacy.com/manual/NiccoMlt/Protelis-Web/pullRequest?prid=4969516)